deploy.Jenkinsfile

```diff
@@ -0,0 +1,97 @@
+                }
+                dir('waves') {
+                    checkout scm: [$class: 'GitSCM', userRemoteConfigs: [[url: 'https://github.com/wavesplatform/waves.git', credentialsId: null]], extensions: [[$class: 'CloneOption', noTags: false, reference: '', shallow: false]], branches: [[name: '*/master']]], poll: false
+
+                }
+            }
+        }
+        stage('Compile Matcher') {
+            steps {
+                dir('matcher') {
+                    sh 'printenv'
+                    sh 'git fetch --tags'
+                    sh 'find ~/.sbt/1.0/staging/*/waves -type d -name target | xargs -I{} rm -rf {}'
+                    sh 'find . -type d -name target | xargs -I{} rm -rf {}'
+                    sh 'sbt "set Global / scalacOptions ++= Seq(\\"-Xfatal-warnings\\", \\"-Ywarn-unused:-imports\\");session save;cleanAll;compile;packageAll"'
+                }
+            }
+        }
+        stage('Compile Node') {
+            steps {
+                dir('waves') {
+                    sh 'find ~/.sbt/1.0/staging/*/waves -type d -name target | xargs -I{} rm -rf {}'
+                    sh 'find . -type d -name target | xargs -I{} rm -rf {}'
+                    sh 'sbt packageAll'
+                }
+            }
+        }
+        stage('Deploy nodes') {
+            parallel {
+                 stage("Deploy node 2") {
+                    steps {
+                        script {
+                            deployNode("${NODE2}").call()
+                        }
+                    }
+                 }
+                 stage("Deploy node 3") {
+                    steps {
+                        script {
+                            deployNode("${NODE3}").call()
+                        }
+                    }
+                 }
+                 stage("Deploy node 4") {
+                    steps {
+                        script {
+                            deployNode("${NODE4}").call()
+                        }
+                    }
+                 }
+            }
+        }
+        stage('Deploy Matcher') {
+            steps {
+                sshagent (credentials: ['buildagent-matcher']) {
+                    sh "ssh -o StrictHostKeyChecking=no -l buildagent-matcher ${MATCHER} hostname"
+                    sh "scp ./matcher/target/release/waves-dex*all*.deb buildagent-matcher@${MATCHER}:/home/buildagent-matcher"
+                    sh "scp ./matcher/target/release/waves-dex-extension-devnet*all*.deb buildagent-matcher@${NODE4}:/home/buildagent-matcher"
+                    sh "scp ./matcher/dex-load/src/main/resources/reinstallExtension.sh buildagent-matcher@${NODE4}:/home/buildagent-matcher"
+                    sh "scp ./matcher/dex-load/src/main/resources/reinstallMatcher.sh buildagent-matcher@${MATCHER}:/home/buildagent-matcher"
+                    sleep time: 80000, unit: 'MILLISECONDS'
```

Comment:
Also here

Reply:
I've gotten the answer upper... may me I try to find a better way to exclude sleep step